ledger/complete/wal/compactor.go

```diff
@@ -111,26 +114,35 @@ func (c *Compactor) createCheckpoints() (int, error) {
 		return -1, fmt.Errorf("cannot get latest checkpoint: %w", err)
 	}
 
+	// if admin tool was used to trigger checkpoint, then it's equivilent to setting checkpoint distance
+	// as 1 for once.
+	if to-from > 1 && c.triggerCheckpointOnNextSegmentFinish.CAS(true, false) {
```

Comment:
I think we should reset the flag in any case - if it was used to trigger a checkpoint or it just happen than next segment would trigger a checkpoint anyway.
In other words - any checkpoint creation should reset this flag